Traceback (most recent call last):
  File "/home/users/sm961/CS+/DeepLabV3Plus-Pytorch/main.py", line 624, in <module>
    main()
  File "/home/users/sm961/CS+/DeepLabV3Plus-Pytorch/main.py", line 606, in main
    save_ckpt('checkpoints/latest_%s_%s_os%d.pth' %
  File "/home/users/sm961/CS+/DeepLabV3Plus-Pytorch/main.py", line 561, in save_ckpt
    "optimizer_state": optimizer.state.dict(),
AttributeError: 'collections.defaultdict' object has no attribute 'dict'
srun: error: linux51: task 0: Exited with exit code 1
